layouts/default/60_ansi_arrow/readme.md

```diff
@@ -1,50 +1,76 @@
-# 60_ansi_arrow
-`LAYOUT_60_ansi_arrow`
+# 60_ansi_arrow and Keymap
```

Comment:
This is a default layout, so it should be made as simple as possible.

Reply:
Ok done.

Reply:
Ok done